{{cookiecutter.repo_name}}/requirements/base.txt

```diff
@@ -34,4 +34,8 @@ django-autoslug==1.8.0
 # Time zones support
 pytz==2015.4
 
+# Redis support
```

Comment:
@pydanny  Memcache requirements were in `/requirements.txt` because heroku needs `pylibmc` at root requirements.txt to install some OS libraries and also `django-heroku-memcachify` was only relevant in context of heroku.